docs/spec_draft.md

```diff
@@ -1410,6 +1411,39 @@ More formally, `result[i0, ..., iR-1]` is equal to:
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.popcnt
+
+### Semantics
+
+Counts the number of bits set in `operand` tensor element-wise and produces a
+`result` tensor.
+
+### Inputs
+
+| Name      | Type                   |
+|-----------|------------------------|
+| `operand` | tensor of integer type |
+
+### Outputs
+
+| Name     | Type                   |
+|----------|------------------------|
+| `result` | tensor of integer type |
+
+### Constraints
+
+  * (C1) `operand` and `result` have the same type.
+
+### Examples
+
+```mlir
+// %operand: [0, 1, 2, 127]
+%result = "stablehlo.popcnt"(%lhs, %rhs) : (tensor<4xi8>) -> tensor<4xi8>
```

Comment:
%lhs %rhs -> %operand

Reply:
%result = "stablehlo.popcnt"(% operand) : (tensor<4xi8>) -> tensor<4xi8>

Reply:
Thanks for the catch!